string room
---
sensor_msgs/Image masked_image
home_robot_msgs/NamedLocation[] receptacles